ReadMe.md

```diff
@@ -0,0 +1,17 @@
+### Readme 
+
+This application will take an old Microsoft Flight Simulator X (FSX) 
+and convert the files and textures to the newer Microsoft Flight Simulator 2020 (MSFS) format
+
+## How To
+
+## Known issues
+* While converter will make these files readable by Microsoft Flight, it is very likely you will need to modify the textures
+  and other resources to make it work properly.
+* Non 3d guages to not always translate over
+
+## To Do
+
+## Prerequisits
+.Net Framework 3.5 development tools 
+(link?)
```

Comment:
```suggestion
## Prerequisites
[.NET Core 3.1 development tools](https://docs.microsoft.com/en-us/dotnet/core/install/windows?tabs=netcore31)
Visual Studio 2019 (optional)
```

Reply:
Updated this, added the link to VS 2019 as well